apps/SparseMatrix/sparse_matrix.cuh

```diff
@@ -0,0 +1,191 @@
+#include <iostream>
+#include <vector>
+#include "rxmesh/attribute.h"
+#include "rxmesh/context.h"
+#include "rxmesh/kernels/query_dispatcher.cuh"
+#include "rxmesh/patch_ptr.h"
+#include "rxmesh/types.h"
+
+namespace rxmesh {
+
+template <uint32_t blockThreads>
+__global__ static void sparse_mat_test(const rxmesh::Context context,
+                                       uint32_t*             patch_ptr_v,
+                                       uint32_t*             vet_degree)
+{
```

Comment:
This function should be moved to be `sparse_matrix_test.cu`. We want to separate the sparse matrix implementation from test it.

Reply:
done